config/webpack/webpack.config.js

```diff
@@ -124,31 +127,45 @@ const buildWebpackConfigs = builds.map(
         return JSON.stringify(valueForEnv);
       })
       .set('SKU_ENV', JSON.stringify(args.env))
-      .set('PORT', JSON.stringify(port))
+      .set('PORT', JSON.stringify(port.length > 1 ? port[1] : ''))
```

Comment:
Need to be PORT or can it be SKU_PORT or similar?

Reply:
Good point, worth updating